frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js

```diff
@@ -237,7 +236,7 @@ class SchemaItem extends React.Component<{
                         key={index}
                         style={{ padding: '5', fontSize: '16px', maxWidth: '100' }}
                     >
-                        {header}
+                        {headerText[header]}
```

Comment:
I think we should still keep saying "generalized-path" and "post-processor", as that is the same terminology we use in the docs: https://codalab-worksheets.readthedocs.io/en/latest/Worksheet-Markdown/#schemas

Reply:
So we'll simply remove the placeholder text but keep the column titles?

Reply:
Actually this issue was brought up last week during the meeting and Percy said it's fine to have two different sets of terminology - this is for clarity's sake to display a more human-readable name on the front end.